CarryUsServer/src/main/java/com/SMWU/CarryUsServer/domain/reservation/controller/response/StoreReviewResponseDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.SMWU.CarryUsServer.domain.reservation.controller.response;
+
+import com.SMWU.CarryUsServer.domain.reservation.entity.ReservationReview;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public record StoreReviewResponseDTO(
+        long reviewId,
+        String memberName,
+        String reviewCreatedAt,
+        double reviewRating,
+        String reviewText
+        ) {
+    public static StoreReviewResponseDTO of(long reviewId, String memberName, String reviewCreatedAt, double reviewRating, String reviewText) {
+        return new StoreReviewResponseDTO(reviewId, memberName, reviewCreatedAt, reviewRating, reviewText);
+    }
+
+    public static StoreReviewResponseDTO toStoreReviewResponseDTO(final ReservationReview reservationReview){
+        return StoreReviewResponseDTO.of(reservationReview.getReservationReviewId(), reservationReview.getReservation().getClient().getName(), getReviewCreatedAt(reservationReview.getCreatedAt()), reservationReview.getReviewRating(), reservationReview.getReviewContent());
+    }
```

Comment:
return값에 너무 많은 get함수가 있는 것 같아서 혹시 이렇게는 어떨까요? (단지 제안입니다!) 

```suggestion
    long reviewId = reservationReview.getReservationReviewId();
    String clientName = reservationReview.getReservation().getClient().getName();
    String reviewCreatedAt = calculateFormattedReviewCreatedAt(reservationReview.getCreatedAt());
    double reviewRating = reservationReview.getReviewRating();
    String reviewContent = reservationReview.getReviewContent();

    return StoreReviewResponseDTO.of(reviewId, clientName, reviewCreatedAt, reviewRating, reviewContent);
```

Reply:
저도 변수 선언과 가독성중에 고민중이긴한데.. 강희님 말씀대로 객체를 통해 받는 값이 .getXX 함수로 많이 이어지고 있어 분리하는게 의미 전달에도 더 좋을 것 같네요 다른 비슷한 부분에도 수정하여 반영하겠습니다!